### DISCUSSION TITLE
How to setup the app without SingPass

### DISCUSSION THREAD

**Discussion by mudassirmaroof** [opengovsg/FormSG#6425] [Q&A] [answered]
As of now it looks like app needs all the SingPass environment variables to be properly configured in order to be able to start the app. If environment variables are missing, are not of correct format like URL etc. or do not point to valid file paths then app throws error while starting up.

How can we configure and run the app such that we do not have to provide any details around SingPass or CorpPass ?

**Comment by LoneRifle** [comment, marked as answer]
If it helps, I have set up a demo instance at https://form.demos.sg, using the config that you can find in https://github.com/demossg/formsg-on-fly . tldr; you can point FormSG to either your own deployment of MockPass, or the MockPass instance that is hosted on Netlify.

If you want to launch your own instance of MockPass, you may wish to consider using https://github.com/demossg/mockpass-on-netlify
